container_files/demos/Virtual manipulation of datasets.ipynb

```diff
@@ -0,0 +1,1583 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Virtual manipulation of datasets tutorial\n",
+    "\n",
+    "In this tutorial, we will be using 2 [Dataset types](../../../../doc/#builtin/datasets/Datasets.md.html), namely [Sampled Dataset](../../../../doc/#builtin/datasets/SampledDataset.md.html) and [Transposed Dataset](../../../../doc/#builtin/datasets/TransposedDataset.md.html). We will be using a manually created dataset that has two columns. The first column contains the name of machine learning concepts. The second column contains the description (i.e. first paragraph written about the concept in [Wikipedia](https://en.wikipedia.org))."
```

Comment:
-> "we will be showing how to use 2 dataset types." JE changerais de paragraphe et qqchose du genre "To use in the examples below, we created a toy dataset using the description of machine learning concepts taken from wiki" bla bla